mona_sdk/mona_single_message.py

```diff
@@ -1,7 +1,18 @@
 import warnings
 from dataclasses import dataclass
+from os import environ
 
-ENABLE_DEPRECATION_WARNINGS_ON_EXPORT_TIMESTAMP = False
+from mona_sdk.client_util import get_boolean_value_for_env_var
+
+MONA_SDK_ENABLE_DEPRECATION_WARNINGS_ON_EXPORT_TIMESTAMP = (
+    get_boolean_value_for_env_var(
+        "MONA_SDK_ENABLE_DEPRECATION_WARNINGS_ON_EXPORT_TIMESTAMP", False
+    )
+)
+
+MONA_SDK_USE_SEND_TIMESTAMP_FIELD = get_boolean_value_for_env_var(
+    "MONA_SDK_USE_SEND_TIMESTAMP_FIELD", False
```

Comment:
add comment what this is and todo to remove after all servers are updated maybe?